webrtc.html

```diff
@@ -78,6 +78,7 @@
     <dfn>MediaStreamConstraints</dfn> are defined in [[!GETUSERMEDIA]].</p>
     <p>The term <dfn>Blob</dfn> is defined in [[!FILEAPI]].</p>
     <p>The term <dfn>media description</dfn> is defined in [[!RFC4566]].</p>
+    <p>The term <dfn>media transport</dfn> is defined in [[!RFC7656]].</p>
```

Comment:
I'm realizing now that "media transport" may not be the right term. This RFC also says "An RTP session uses at least two media transports: one for sending and one for receiving." I'm not sure if this is just because it was written with unidirectional transports in mind (5-tuples), or if a media transport is fundamentally intended to be unidirectional.

Also, who's to say that a new DTLS association is not a new media transport? There's nothing that defines whether that's the case or not.

Really, I'd like a term that means "the transport layer of a media description, which (using BUNDLE) multiple media descriptions may share". But I don't know if that term already exists.

Reply:
The term that ICE uses is "component", I found when doing that review.